### PR TITLE
Elements: define interactibility

### PR DESCRIPTION
**Pull request by andreastt**
Based on resolution from
http://www.w3.org/2015/10/25-webdriver-minutes.html.

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -2882,6 +2888,68 @@
  whose <a><code>readOnly</code> attribute</a> is set.
 
 <section>
+<h3>Element Interactibility</h3>
+
+<p>In order to determine if an element can be interacted with,
+ WebDriver performs hit-testing to find
+ if the interaction will be able to reach the requested element.
```
Comment (AutomatedTester) [inline webdriver-spec.html]: This is only true for elements we are using pointer actions on.

File: webdriver-spec.html [hunk 2]
```diff
@@ -2882,6 +2888,68 @@
...
+
+<p>An <dfn>interactible element</dfn> is defined to be
+ the first non-transparent <a>element</a>,
```
Comment (AutomatedTester) [inline webdriver-spec.html]: what is `non-transparent`?

File: webdriver-spec.html [hunk 3]
```diff
@@ -2882,6 +2888,68 @@
...
+  because elementsFromPoint takes care of it
+
+  this is the same as
+
+    rect.width - (rect.right - innerWidth)
+    rect.height - (rect.bottom - innerHeight)
+ -->
+ <li><p>Let <var>visible width</var> be
+  the <var>element</var>’s <a>bounding rectangle</a>’s
+  (<a>max</a>(<a>x coordinate</a>, <a>x coordinate</a> + <a>width dimension</a>) –
```
Comment (AutomatedTester) [inline webdriver-spec.html]: what is `max`?

File: webdriver-spec.html (hunk 2 above)
Comment (andreastt) [inline webdriver-spec.html]: This is just the non-normative description, but it means `opacity` set to != "1".

File: webdriver-spec.html (hunk 3 above)
Comment (andreastt) [inline webdriver-spec.html]: See the definition under _Terminology_.  It’s a way to get the largest value of a variable list of values.

**Comment by AutomatedTester** [comment]
landed in 29ce8d5

**Pull request closed without merging**